src/main/java/com/disraely/entities/Employee.java

```diff
@@ -0,0 +1,28 @@
+package com.disraely.entities;
+
+import com.disraely.enums.EmployeeLevel;
+
+public abstract class Employee extends BaseEntity {
+    private Double salary;
```

Comment:
I don't think this is necessary in this moment

Reply:
Resolved!